src/Rules/UnusedPublicClassMethodRule.php

```diff
@@ -112,7 +112,10 @@ private function isUsedClassMethod(
             return true;
         }
 
+        // php method calls are case-insensitive
+        $completeMethodCallReferences = array_map(fn(string $item) => strtolower($item), $completeMethodCallReferences);
```

Comment:
On the left should be different variable name, as different content. E.g. lowercasedMethodCallNames

Reply:
here you are